jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -5758,6 +5758,104 @@
 				}
 			]
 		},
+		{
+			"name": "p) create JVM with bad credentials",
+			"description": "",
+			"item": [
+				{
+					"name": "create jvm with bad user credentials",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "bbc6d7d0-ceec-4469-926a-a12be613b4b4",
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 201\"] = responseCode.code === 201;",
+									"var jsonData = JSON.parse(responseBody);",
+									"postman.setEnvironmentVariable(\"BadJvmId\", jsonData.applicationResponseContent.id.id);"
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"jvmName\":\"test-jvm-failed\",\"groupIds\":[{\"id\":10,\"groupId\":{{GroupId}}}],\"hostName\":\"{{host}}\",\"statusPath\":\"/tomcat-power.gif\",\"systemProperties\":\"\",\"httpPort\":\"8000\",\"httpsPort\":\"8001\",\"redirectPort\":\"8002\",\"shutdownPort\":\"8003\",\"ajpPort\":\"8004\",\"userName\":\"baduser\",\"encryptedPassword\":\"bad\",\"jdkMediaId\":{{JdkMediaId}},\"tomcatMediaId\":{{TomcatMediaId}}}"
```

Comment:
Is it possible to not hard code the group ID?
We'll get errors in the test if we run them back-to-back and the group ID keeps incrementing.